app/assets/javascripts/controllers/cloud_volume/cloud_volume_form_controller.js

```diff
@@ -213,6 +213,8 @@ ManageIQ.angular.app.controller('cloudVolumeFormController', ['miqService', 'API
     // model attribute with AWS.
     vm.cloudVolumeModel.aws_volume_type = data.volume_type;
     vm.cloudVolumeModel.aws_availability_zone_id = data.availability_zone.ems_ref;
+    vm.cloudVolumeModel.aws_encryption = data.encrypted;
+    vm.cloudVolumeModel.aws_iops = data.iops;
```

Comment:
@AparnaKarve the code that you have referenced should probably not be changed as we still want to allow IOPS to be dynamically calculated whenever the size changes. However, we should review [this](https://github.com/ManageIQ/manageiq-ui-classic/blob/master/app/assets/javascripts/controllers/cloud_volume/cloud_volume_form_controller.js#L224) as the sole purpose of this invocation is to calculate initial IOPS based on the size.

@sasoc Please, test the behaviour. If IOPS can be undefined, then call `vm.sizeChanged` only when iops is undefined. If it's defined, skip it.